.github/workflows/ci.yml

```diff
@@ -27,13 +27,22 @@ jobs:
         NXF_VER:
           - "23.04.0"
           - "latest-everything"
+        tags:
+          - "test"
+          - "test_krakenuniq"
+          - "test_malt"
+          - "test_motus"
+          - "test_nopreprocessing"
+          - "test_noprofiling"
+          - "test_nothing"
         parameters:
-          - "--preprocessing_qc_tool falco"
-          - "--shortread_qc_tool fastp"
-          - "--shortread_qc_tool adapterremoval"
-          - "--shortread_complexityfilter_tool bbduk"
-          - "--shortread_complexityfilter_tool prinseqplusplus"
-
+          - "--preprocessing_qc_tool=falco"
+          - "--shortread_qc_tool=fastp"
+          - "--shortread_qc_tool=adapterremoval"
+          - "--shortread_complexityfilter_tool=bbduk"
```

Comment:
I guess these still need to be replaced with profiles?

Reply:
Do you mean changed it into: 
` - "docker,--preprocessing_qc_tool=falco,--shortread_qc_tool=fastp,--shortread_qc_tool=adapterremoval,--shortread_complexityfilter_tool=bbduk,--shortread_complexityfilter_tool=prinseqplusplus"
`

Reply:
No, if i have a spare 30 minutes today I'll push changes what I mean

Reply:
What I mean is: `-profile test_falco` for example

Reply:
Aha, so we also create a test_falco.config?